src/tasks.py

```
@@ -0,0 +1,148 @@
+from src.services.genai import GenAIOrchestrator
+from src.services.openai_image_client import OpenAIImageClient
+from src.services.processor import ImageProcessor
+from src.services.storage import StorageManager
+from src.utils.config import settings
+from src.utils.logger import get_logger
+
+logger = get_logger(__name__)
+
+
+@celery_app.task(bind=True, name="process_campaign_task")
+def process_campaign_task(self, brief_dict: Dict[str, Any]) -> Dict[str, Any]:
```

Ruff (0.13.3)

Unused function argument: `self`

(ARG001)